conjur/data_object/conjurrc_data.py

```diff
@@ -57,7 +58,8 @@ def load_from_file(cls, conjurrc_path: str = DEFAULT_CONFIG_FILE):
                                     loaded_conjurrc.get('account') or loaded_conjurrc['conjur_account'],
                                     loaded_conjurrc['cert_file'],
                                     loaded_conjurrc.get('authn_type'),
-                                    loaded_conjurrc.get('service_id'))
+                                    loaded_conjurrc.get('service_id'),
+                                    loaded_conjurrc.get('use_netrc'))
```

Comment:
I like this idea, just pushed up the changes. 

Regarding your first comment on `self.netrc_path`, I feel as though using the `netrc_path` key implies that the netrc location should be configurable, in which case I would probably leave alone the instance var since it supports that notion (even though it is not fully implemented yet)